lib/js/components/SelectionTile/SelectionTile.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+	<div
+		class="selectionTile"
+		:class="[
+			{
+				'-selected': isSelected,
+				'-default': state === SELECTION_TILE_STATE.DEFAULT,
+				'-disabled': state === SELECTION_TILE_STATE.DISABLED,
+				'-loading': state === SELECTION_TILE_STATE.LOADING,
+			},
+		]"
+		@click="updateIsSelected(!isSelected)"
+	>
+		<div class="selectionTile__wrapper">
+			<div class="selectionTile__selectionControl">
+				<radio-button
+					v-if="type === SELECTION_TILE_TYPE.RADIO_BUTTON"
+					:size="RADIO_BUTTON_SIZE.X_SMALL"
+					:is-selected="isSelected"
+					:state="selectionControlState"
+					@update:isSelected="updateIsSelected($event)"
+				/>
+				<checkbox
+					v-else
+					:size="CHECKBOX_SIZE.X_SMALL"
+					:is-selected="isSelected"
+					:state="selectionControlState"
+					@update:isSelected="updateIsSelected($event)"
+				/>
+			</div>
+			<div class="selectionTile__textWrapper">
+				<div class="selectionTile__title">{{ title }}</div>
+
+				<div v-if="supportingText" class="selectionTile__supportingText"
+					>{{ supportingText }}
+				</div>
+			</div>
+		</div>
+		<div
+			v-if="icon || isLoading"
+			class="selectionTile__icon"
+			@click.stop="isLoading ? null : $emit('icon-click')"
```

Comment:
Minor issue, I found this because I was experimenting with storybok Actions Tab
because of `.stop` here, `click` event isn't fired on entire component. I'd expect `click` to be fired here as well...

Reply:
stop here is for stopping bubbling when clicking this icon, without this after click in icon (eg to open filters) we're toggling selection tile, i changed it to no stopping when loading

Reply:
ok, let's keep it as it is. I hope noone uses `click` event handler on this component :see_no_evil: